cvcl/views/course.py

```diff
@@ -56,4 +56,4 @@ class CourseDelete(LoginRequiredMixin, SuccessMessageMixin, DeleteView):
     success_message = "%(name)s was deleted successfully"
 
     def get_queryset(self):
-        return Course.objects.filter(instructor=self.request.user)
+        return Course.objects.filter(instructor=self.request.user)
```

Comment:
See your detailed diff. You removed "newline" from end of file. Don't do that, all files should end with a newline.